discovery-provider/src/queries/notifications.py

```diff
@@ -196,6 +196,41 @@ def get_cosign_remix_notifications(session, max_block_number, remix_tracks):
 
     return remix_notifications
 
+def get_prev_track_entries(session, entries):
```

Comment:
Love this query. Perhaps we should just move this out in to src/queries as it's own `get_prev_track_entries.py` file and also give it a doc-string explaining the intent

```
def get_prev_track_entries(session, entries):
   """
   Gets the previous state of a track in the database given a track id and a target blocknumber.
    For example, ...
   """